packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.html

```diff
@@ -25,3 +14,14 @@
     <div class="nui-step-optional" *ngIf="optional && state != 'error'">{{optionalLabel}}</div>
     <div class="nui-step-sub-label-error" *ngIf="state == 'error'">{{errorMessage}}</div>
 </div>
+
+<div class="nui-step-icon-state-{{state}} nui-wizard-step-icon"
+     [class.nui-wizard-step-header__icon--selected]="selected">
+    <div class="nui-step-icon-content">
+        <span class="nui-icon-container">
```

Comment:
It seems like the container class is inside the content class instead of the other way around. I would expect it to be:

```
<div class="container"> 
    <span class="content"></span>
</div>
```